### DISCUSSION TITLE
Structure of simulation

### DISCUSSION THREAD

**Discussion by lukne541** [riebl/artery#241] [Q&A]
Is there documentation for which files that are required for a simulation and what these files should contain? I've looked through artery's documentation and haven't found anything.

**Comment by riebl** [comment]
Just as any other OMNeT++ simulation, Artery requires an *omnetpp.ini* file to set up and configure the simulation environment.
Additional files are depicted in http://artery.v2x-research.eu/architecture/ as grey boxes:
- SUMO needs **.net.xml* (the road network), **rou.xml* (the traffic on the road network), and optionally **.poly.xml* (shape of buildings and other static obstacles on the map). Please refer to the SUMO documentation for details.
- Artery's V2X services are configured via *services.xml*, which is referenced in the *omnetpp.ini*
- If you use Artery's environment model, *sensors.xml* configures the sensors attached to vehicles
- If you use Artery's storyboard feature, a Python script is loaded, modifying the simulation dynamically. The name of the file is specified in the *omnetpp.ini* again.

I hope this helps!
